addons/l10n_pe_pos/static/src/overrides/components/ticket_screen/ticket_screen.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module */
+
+import {TicketScreen} from "@point_of_sale/app/screens/ticket_screen/ticket_screen";
+import {patch} from "@web/core/utils/patch";
+
+patch(TicketScreen.prototype, {
+    setPartnerToRefundOrder(partner, destinationOrder) {
```

Comment:
Why do you have two consecutive nested `if` statements? Isn't better to put the whole evaluation within the same `if`?

Something like:
```
if (!peruvian) {
    return  super.setPartnerToRefundOrder(...arguments);
}
return destinationOrder.set_partner(partner);